content/100-getting-started/02-setup-prisma/100-start-from-scratch/120-mongodb/200-install-prisma-client.mdx

```diff
@@ -19,7 +19,7 @@ Notice that the install command automatically invokes `prisma generate` for you
 
 ![Install and generate Prisma Client](./../../../images/prisma-client-install-and-generate.png)
 
-Whenever you make changes to your Prisma schema in the future, you manually need to invoke `prisma generate` in order to accommodate the changes in your Prisma Client API.
+Whenever you update your Prisma schema, you will need to regenerate Prisma Client using the `prisma generate` command to update your Prisma Client API.
```

Comment:
I'm actually not sure about the recommended workflow on MongoDB but I thought you were supposed to run `prisma db push` (which would trigger `generate`), or do we always recommend to just run `prisma generate`?

Reply:
Seems this workflow is also not present in the getting started guide from scratch. I think `prisma db push` creates unique and id fields. 😅

Reply:
> `prisma generate` should work but I'll confirm it before merging the PR

Yeah, the question is what is best practice is that we want to recommend to our users and my understanding was that `prisma db push` should be used. (I think we should strive to make it as simple as possible for them and not teach them two approaches that do the same thing but teach one approach that should be default and then maybe at some point they can learn about a different way to achieve the same goal. ).

Reply:
It makes sense to have the `prisma db push` command.